corehq/apps/export/tests/test_export_forms.py

```diff
@@ -261,9 +261,10 @@ def test_get_user_type_filter_for_active_and_deactivated_mobile(self, fetch_user
         self.assertIsInstance(user_filters[0], UserTypeFilter)
         self.assertEqual(user_filters[0].user_types, self.subject._USER_MOBILE)
 
-    @patch.object(form, '_get_selected_es_user_types', return_value=[HQUserType.ACTIVE, HQUserType.DEACTIVATED, HQUserType.ADMIN])
+    @patch.object(form, '_get_selected_es_user_types', return_value=[HQUserType.ACTIVE, HQUserType.DEACTIVATED,
+                                                                     HQUserType.ADMIN])
     @patch.object(filter_builder, 'get_user_ids_for_user_types')
-    def test_get_user_type_filter_for_admin_and_deactivated_and_active_mobile(self, fetch_user_ids_patch, *patches):
+    def test_get_user_type_filter_for_admin_and_deactivated_and_active_mobile(self, fetch_user_ids_patch):
```

Comment:
FYI, if you provide a `new` argument to your `@patch` call, then it doesn't pass in the mock to the method ([docs](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch)).

Reply:
Didn't know that, thanks! https://github.com/dimagi/commcare-hq/pull/21564/commits/ede5c8e7999fff3fcdd41925e00328bd5802d5a5